db/db_test.go

```diff
@@ -45,8 +44,5 @@ func TestDB(t *testing.T) {
 		}
 		err = Close(table)
 		So(err, ShouldBeNil)
-
-		table, err = Open(testFile)
-		So(err, ShouldBeNil)
```

Comment:
This should not be removed, it's a test to reopen closed db.

Reply:
Ah ok.
Sorry for that!